jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java

```diff
@@ -264,8 +265,9 @@ private void prepareDocs(Map<String, Map<String, String>> currentInfo, List<Map<
             } else if (entityInfo.containsKey("projectId")) {
                 entityInfo.put("accountid", entityInfo.get("projectId"));
             }
-            //For GCP CQ Collector accountName will be fetched from RDS using accountId
-            if ("gcp".equalsIgnoreCase(dataSource)) {
+            // For GCP CQ Collector accountName will be fetched from RDS using accountId
```

Comment:
this is CE code, can't expose CQ logic here. anyway to make this generic?

Reply:
probably just keep the comment for now.

Reply:
I future yup, it should and will be generic. I guess, after we redefine how we work with accounts. All of those should have same naming for IDs and names, I see it like `accountID` and `accountName` for all types of sources, not like we have now - every single data source has its own naming for those.

Reply:
fixed